objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -711,21 +856,64 @@ public void testQuery() throws ImageProcessingException, InterruptedException {
             + "\"right\":350.1204}, \"label\":\"keyboard\"}, {\"confidence\":0.7974271, \"location\":{\"top\":91.255974, \"left\":164.41359, "
             + "\"bottom\":275.69666, \"right\":350.50714}, \"label\":\"tvmonitor\"}]";
     
-    String neuralNetJSON = 
+    String neuralNetJSON1 = 
               "{"
             + "     \"neuralNet\":"
             + "         {"
             + "             \"anchors\":[0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828], "
             + "             \"pbFile\": \"" + PB_FILE + "\", "
             + "             \"labelFile\": \"" + LABEL_FILE + "\", "
+            + "             \"metaFile\": \"" + META_FILE + "\", "            
             + "             \"type\": \"yolo\""
             + "         }"
             + "}";
+    
+    String neuralNetJSON2 = 
+            "{"
+          + "     \"neuralNet\":"
+          + "         {"
+          + "             \"pbFile\": \"" + PB_FILE + "\", "
+          + "             \"metaFile\": \"" + META_FILE + "\", "            
+          + "             \"type\": \"yolo\""
+          + "         }"
+          + "}";
+    
+    String neuralNetJSON3 = 
+            "{"
+          + "     \"neuralNet\":"
+          + "         {"
+          + "             \"anchors\":[0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828], "
+          + "             \"pbFile\": \"" + PB_FILE + "\", "
+          + "             \"labelFile\": \"" + LABEL_FILE + "\", "
+          + "             \"type\": \"yolo\""
+          + "         }"
+          + "}";
+    
+    String neuralNetJSON4 = 
+            "{"
+          + "     \"neuralNet\":"
+          + "         {"
+          + "             \"pbFile\": \"" + PB_FILE + "\", "
+          + "             \"labelFile\": \"" + LABEL_FILE + "\", "
+          + "             \"type\": \"yolo\""
+          + "         }"
+          + "}";
 
     List<Map> getExpectedResults() throws JsonParseException, JsonMappingException, IOException {
         ObjectMapper m = new ObjectMapper();
         return m.readValue(imageResultsAsString, List.class);
     }
+    
+    void verifyProcessing(YoloProcessor ypImageSaver) throws ImageProcessingException {
+        NeuralNetResults results = ypJson.processImage(getTestImage());
```

Comment:
Umm, this ignores the input parameter.  So still doesn't test things...

Reply:
Oh man I'm sorry that was a pretty dumb mistake. I'm fixing it now.